README.md

```diff
@@ -11,11 +11,11 @@ In light of recent developments, we have introduced a series of RPMs web develop
 - [Container Contribution Guidelines](contribution.md): Learn about what principles to follow when contributing to the SCLOrg, particularly in relation to container repositories.
 - [Container Images Overview Based on Software Collections](images.md): Access Dockerfiles, relevant sources, and tests for container images.
 
-- [scl-utils](https://github.com/sclorg/scl-utils): The upstream project for the scl-utils package available in RHEL, CentOS, and Fedora, enabling the building of RPM packages in an alternative directory.
+- [scl-utils](https://github.com/sclorg/scl-utils): The upstream project for the scl-utils package available in RHEL, CentOS Stream and Fedora, enabling the building of RPM packages in an alternative directory.
```

Comment:
Just remove all mention of scl and centos at this point tbh.

Reply:
Removed